NewForce-at-Mountwest/bangazon-site-the-blue-ferrets
# AC-Bangazon

# Description

Given a user has filled out the product form, but not chosen a category
When the user clicks on the Sell button
Then the user should be alerted to select a product category

Fixes # (issue)

When the user clicks on the Sell button
Then the user should be alerted to select a product category

## Type of change
error message given when product category not selected

Please delete options that are not relevant.




# Testing Instructions

-Git add and git commit current branch
-Git checkout master
-Git fetch --all
-Git checkout AC-Bangazon
-Launch browser with debugger or ctrl F5
-Logout if necessary
-Log in as: admin@admin.com
-Password:  Admin8*
-Please click affordance Sell Something on NavBar
-Click Create New
-fill out product details form in full 
-DO NOT select a product category
-select save
-a prompt to choose a product category will appear in red and not allow user to continue

# Checklist:
